dbt_project.yml

```diff
@@ -16,4 +17,5 @@ vars:
         campaign_group_history: "{{ source('linkedin','campaign_group_history') }}"
         campaign_history: "{{ source('linkedin','campaign_history') }}"
         creative_history: "{{ source('linkedin','creative_history') }}"
-        
+    
+    linkedin__passthrough_metrics: [] 
```

Comment:
Have we been using this naming convention with variables? If not, I think we should start to standardize our variables to reflect this as it will also help users differentiate package variables at a global hierarchy - and also for a package like this one where it does roll up to the ads roll up package.

Reply:
yeah i was just keeping consistent with the pre-existing `linkedin__use_local_currency` variable. I personally don't usually use double underscores for variables 🤷

Reply:
but in terms of prefixing variables with the package name, yah i've been doing that! most variables need to be global/across the source and transform packages anyways it seems